Violet Rails/Common/Constants/Link.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Link.swift
+//  Violet Rails
+//
+//  Created by Joses Solmaximo on 31/03/23.
+//
+
+import Foundation
+
+struct Link {
```

Comment:
@josessolmaximo could we rename `Link` to `App` and `ROOT` to `ENDPOINT`

Reply:
Sure